etc/tempesta_fw.conf

```diff
@@ -92,6 +92,14 @@
 # Default:
 #   None.
 
+cache 0;
+
+server 10.55.1.1:80;
+
+block_action error reply;
+
+block_action attack reply;
+
```

Comment:
Seems a mistaken change: the file is used by end users, so never put any testing stuff here, especially, specific for your local installation.

Reply:
fixed